portfolio/src/main/webapp/index.html

```diff
@@ -30,6 +30,19 @@ <h1>Heidi Johnson's Portfolio</h1>
             <input type="submit" />
           </form>
           <ul id="comments-container"></ul>
+          
+          <label for="number-comments">Number of comments:</label>
+            <select onchange="getNumCommentsSelected(this); storeSelectedValue(this)" name="number-comments" id="number-comments">
```

Comment:
You should always wrap functionality like this in a single method call instead of writing out multiple expressions in an `onchange` etc. field, so here you should implement a single method to call rather than call both `getNumCommentsSelected` and `storeSelectedValue` separately.